src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -268,7 +239,8 @@ function useNewlyCompletedTransactions(
   useEffect(
     function updatePrevStandBy() {
       setPreviousStandBy((prev) => {
-        const confirmedHashes = standByTransactions.confirmed.map((tx) => tx.transactionHash)
+        const { pending, confirmed } = categorizeTransactions(standByTransactions)
+        const confirmedHashes = confirmed.map((tx) => tx.transactionHash)
```

Comment:
nit: do we need to loop through to get transaction hashes, and then loop through again to use the `.includes` below? could we instead do `confirmed.some(({transactionHash}) => transactionHash === tx.transactionHash)` to get `newlyCompleted`?

Reply:
@kathaypacific you're right about replacing `.filter` with `.some`! For some reason I interchangeably use both when I do these checks and I don't really know why I always pick one or the other 😄 

I'll still need to `.map` for hashes whether for `confirmed` or `prev.prending` cause otherwise I'll need to do:
```ts
confirmed.some(({transactionHash}) => {
  const pendingTx = prev.prending.find((tx) => tx.transactionHash);
  return pendingTx && transactionHash === pendingTx?.transactionHash
}) 
```

Reply:
@kathaypacific Oh, I've remember why I made it with `.filter`.
`newlyCompleted` is also used to calculate `newlyCompletedCrossChainSwaps`